main.go

```diff
@@ -1,18 +1,18 @@
 package main
 
 import (
-	"encoding/json"
 	"fmt"
```

Comment:
Missing Test: consider adding unit tests for `json.Unmarshal` and `json.Marshal`